app/build.gradle

```diff
@@ -68,6 +68,7 @@ dependencies {
     androidTestImplementation 'androidx.test:rules:1.3.0'
     androidTestImplementation 'androidx.test.espresso:espresso-intents:3.3.0'
     androidTestImplementation 'androidx.test.ext:junit:1.1.2'
+    androidTestImplementation 'androidx.test.espresso:espresso-contrib:3.2.0'
```

Comment:
I think that ligns 71 and 69 shouldn't be both there. I would be inclined to say the 3.3 should stay since it was previously there

Reply:
I am not sure I understand, they are not the same imports, I needed the espresso-contrib to implement the tests for the sliding menu.

Reply:
Oh sorry my mistake